lib/utils/jsontools.go

```diff
@@ -100,6 +101,16 @@ func FastMarshalIndent(v interface{}, prefix, indent string) ([]byte, error) {
 	return data, nil
 }
 
+// WriteJSON marshals multiple documents as a JSON list with indentation.
+func WriteJSON(w io.Writer, values interface{}) error {
+	data, err := json.MarshalIndent(values, "", "    ")
```

Comment:
Can you use an encoder that writes directly to the file rather than marshaling it all into a slice and then writing the slice to the file?

For small collections what you have here is fine, but for larger collections it will require a lot more memory.

Reply:
Thank you for the suggestion. Done in 425ca9bb2b763c2e0d159c4cc80fc33363c12956.